cpp/include/cudf/detail/null_mask.cuh

```diff
@@ -171,24 +182,28 @@ size_type inplace_bitmask_binop(
   return d_counter.value(stream);
 }
 
+enum class count_bits_policy : bool {
```

Comment:
I added a `@brief`. Can you explain where you would put the link? I am still fairly new to doxygen and haven't actually built the C++ docs myself yet.

Reply:
An `@brief` should more than suffice. Thank you for making that change. Since the enum is user-facing, this should help.

FWIW, I believe `@link` is the `javadoc` equivalent of doxygen's `\ref`. The link should be automatically inserted, were we to mention that `count_bits_policy` is used in `cudf::segmented_count_set_bits()`.